[IMP] Inventory: Starshipit update v17 (#18916)
odoo/documentation: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/starshipit_shipping.rst

Reviewer: ```suggestion
Starshipit setup
```

Reviewer: ```suggestion
- :guilabel:`Shipping Method`: Enter `Starshipit` in this field.
```
Just to make it a full sentence ot match the others.

Reviewer: ```suggestion
- :guilabel:`Provider`: Select :guilabel:`Starshipit` from the drop-down menu.
```

Reviewer: ```suggestion
Doing so opens the *Choose Starshipit Shipping Service* pop-up window. In the
```

Reviewer: ```suggestion
The chosen delivery service will populate the :guilabel:`Service Name` field.
```

Reviewer: ```suggestion
In the *Add a shipping method* pop-up window, select Starshipit in the :guilabel:`Shipping
```

Reviewer: Also, if 'Starshipit' is a selectable item from a drop-down, it should have a GUI (and if it's typed in, then the " ` ` " marks).

Starshipit shipping
===================

Starshipit is a shipping service operator that facilitates the integration of Australasian shipping

> couriers with Odoo. Once integrated, users can create shipping methods that will automatically get

Reviewer: ```suggestion
couriers with Odoo. Once integrated, users can create shipping methods that will automatically calculate
```

> rates from specific couriers (such as Australia Post, NZ Post, DHL,...) based on predefined

Reviewer: ```suggestion
rates from specific couriers (such as Australia Post, NZ Post, DHL, etc.) based on predefined
```
Just a preference - nothing wrong with oomlats (umlats?)

-------

After the Starshipit account is set up, integrate it with the Odoo database. To do that, go to

> Odoo's :menuselection:`Apps` module, search for the :guilabel:`Starshipit Shipping` module, and

Reviewer: ```suggestion
Odoo's :menuselection:`Apps` module, search for  `Starshipit Shipping`, and
```
Justbecause that's what they'll type in the search bar

> - :guilabel:`Delivery Product`: assign or create the delivery product that will appear on the sales

Reviewer: ```suggestion
- :guilabel:`Delivery Product`: Assign or create the delivery product that will appear on the sales
```

  order line when the cost of shipping is computed.

.. note::
   The fields discussed in this section are specific to configuring Starshipit. For more information
   about the other fields, refer to :doc:`../setup_configuration`.

> In the :guilabel:`Starshipit Configuration` tab, fill out these fields:

Reviewer: ```suggestion
In the *Starshipit Configuration* tab, fill out these fields:
```
Tabs get italics

> - :guilabel:`Starshipit API Key`: enter the :abbr:`API (Application Programming Interface)` key

Reviewer: ```suggestion
- :guilabel:`Starshipit API Key`: Enter the :abbr:`API (Application Programming Interface)` key
```
Seems like a full sentence (plus I like ot match lists, all sentence,s or all nothing =D )

  obtained from Starshipit.
- :guilabel:`Starshipit Subscription Key`: enter the subscription key obtained from the same place
  as the API key.
- :guilabel:`Origin Address`: Enter the address where products are shipped from. This field is
  crucial for calculating shipping rates and :ref:`generating shipping labels
  <inventory/shipping_receiving/star-label>`.
- :guilabel:`Default Package Type`: Set a default package type to include the weight of the empty
  package when automatically calculating shipping rates.

.. important::
   To set a default package type, the :guilabel:`Packages` feature **must** be enabled in
   :menuselection:`Inventory --> Configuration --> Settings`.

Manually :guilabel:`Save` the form by clicking the cloud icon next to the :guilabel:`Shipping

> Methods / New` breadcrumbs.

Reviewer: ```suggestion
Manually :guilabel:`Save` the form by clicking the :icon:`fa-cloud-upload` :guilabel:`(Save manually)` button.
```

   | :guilabel:`Starshipit Service Code`: `STANDARD-DROPOFF`

.. image:: starshipit_shipping/starshipit-configuration.png
   :alt: Example of shipping products configured in Odoo.

> .. tip::

Reviewer: ```suggestion
.. important::
```
Since thi deals with finances, I think this is an important block (as in, they can be charged for a ton of labels if a poorly trainedperson was following this and maybe didn't cancel, etc)

   Odoo has a built-in layer of protection against unwanted charges when using test environments.
   Within a test environment, if a shipping method is used to create labels, then those labels are
   immediately canceled after creation — this occurs automatically. Please note that depending on
   the shipping provider being used, the account might be charged for printing label, unless the
   order is cancelled manually on the couriers’s portal.

> 

Reviewer: ```suggestion
   automatically canceled after creation.  Depending on the shipping provider being used, the account might be charged for printing a label, unless the order
   is canceled manually on the couriers's portal.
```

Method` field.

Calculate the shipping rate by clicking :guilabel:`Get rate`.

Finally, click :guilabel:`Add` to include the cost of shipping to the sales order line, labeled as
the *delivery product*.

> .. note::

Reviewer: ```suggestion
.. tip::
```
Feels more like a tip?

>    Automatically calculate shipping costs for Starshipit in *both* Odoo **Sales** and **eCommerce**

Reviewer: ```suggestion
   Odoo automatically calculates shipping costs for Starshipit in *both* the **Sales** and **eCommerce**
```

   Verify the destination address, as Starshipit checks it when the order is created.

   Finally, some couriers may require other information, such as an email address or phone number.
   Ensure that all necessary information is set upon sending a shipping order.

Returns
-------

Starshipit allows returns with the following couriers:

- Australia Post eParcel
- TNT
- Couriers Please
- Aramex
- StarTrack
- DHL Express
- NZ Post Domestic

To return products, click the :guilabel:`Return` smart button on the intended delivery order. If the
selected courier supports returns, the :guilabel:`Print Return Label` button is available.

Cancellations
-------------

> If a delivery order is cancelled in Odoo, it will be automatically archived in Starshipit. However,

Reviewer: ```suggestion
If a delivery order is canceled in Odoo, it will be automatically archived in Starshipit. However,
```
I just hate British English, so unless there is a button with the wrong (yes, I said wrong, not different) spelling, then I always defer to the US spelling. One of my apps spells canceled as "Cancelled" in a button, or on a form, so I have to match it, and I hate it every time. This is totally optional, but I will always make this suggestion when I see it. =)
